pytorch_toolkit/formula_recognition/tests/test_export.py

```diff
@@ -0,0 +1,124 @@
+            self.assertEqual(pred_onnx, pred_pytorch)
+
+        def test_4_encoder_ir_export(self):
+            if self.config.get("export_ir"):
+                encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'xml')
+                result_model_exists = os.path.exists(encoder_res_name)
+                if result_model_exists:
+                    os.remove(encoder_res_name)
+                encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'bin')
+                result_model_exists = os.path.exists(encoder_res_name)
+                if result_model_exists:
+                    os.remove(encoder_res_name)
+
+                self.exporter.export_encoder_ir()
+                encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'bin')
+                result_model_exists = os.path.exists(encoder_res_name)
+                self.assertEqual(True, result_model_exists)
+                encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'xml')
+                result_model_exists = os.path.exists(encoder_res_name)
+                self.assertEqual(True, result_model_exists)
+
+        def test_5_decoder_ir_export(self):
+            if self.config.get("export_ir"):
+                decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'xml')
+                result_model_exists = os.path.exists(decoder_res_name)
+                if result_model_exists:
+                    os.remove(decoder_res_name)
+                decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'bin')
+                result_model_exists = os.path.exists(decoder_res_name)
+                if result_model_exists:
+                    os.remove(decoder_res_name)
+
+                self.exporter.export_decoder_ir()
+                decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'bin')
+                result_model_exists = os.path.exists(decoder_res_name)
+                self.assertEqual(True, result_model_exists)
+                decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'xml')
+                result_model_exists = os.path.exists(decoder_res_name)
+                self.assertEqual(True, result_model_exists)
+
+        def test_6_run_ir_model(self):
+            if self.config.get("export_ir"):
+                targets_ir = self.exporter.run_ir_model()
+                ir_pred = self.exporter.vocab.construct_phrase(targets_ir)
+                _, targets_pytorch = self.exporter.model(self.exporter.img)
+                pred_pytorch = self.exporter.vocab.construct_phrase(targets_pytorch[0])
+                self.assertEqual(ir_pred, pred_pytorch)
+    return TestExport
+
+
+class TestMediumRenderedExport(create_export_test_case(CONFIGS[0])):
+    "Test case for medium config"
+
+
+class TestHandwrittenPolynomialsExport(create_export_test_case(CONFIGS[0])):
```

Comment:
1?

Reply:
Probably it will be more explicit if you write config name explicitly. Do not why do you need CONFIGS at all.

Reply:
Removed `CONFIGS` variable